compiler/prelude/genmin.go

```diff
@@ -0,0 +1,60 @@
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"go/build"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/gopherjs/gopherjs/compiler/prelude"
+)
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatalln(err)
+	}
+}
+
+func run() error {
+	bpkg, err := build.Import("github.com/gopherjs/gopherjs", "", build.FindOnly)
+	if err != nil {
+		return fmt.Errorf("failed to locate path for github.com/gopherjs/gopherjs: %v", err)
+	}
+
+	preludeDir := filepath.Join(bpkg.Dir, "compiler", "prelude")
+
+	args := []string{
+		filepath.Join(bpkg.Dir, "node_modules", ".bin", "uglifyjs"),
+		"--config-file",
+		filepath.Join(preludeDir, "uglifyjs_options.json"),
+	}
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Stdin = strings.NewReader(prelude.Prelude)
+
+	out, err := cmd.CombinedOutput()
```

Comment:
I think it is fragile to assume there is no stderr when err is not found. Isn't just `Output` enough?